AppSyncRealTimeClient/Support/AppSyncURLHelper.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2018-2021 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public enum AppSyncURLHelper {
```

Comment:
See this comment from Android regarding referencing some AppSync docs https://github.com/aws-amplify/amplify-android/pull/1564/files#r746083792

I think we should do the same here